app/src/main/java/com/android/aniviewer_kun/MediaRowAdapter.kt

```diff
@@ -2,22 +2,25 @@ package com.android.aniviewer_kun
 
 import android.view.LayoutInflater
 import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.findFragment
 import androidx.recyclerview.widget.DiffUtil
 import androidx.recyclerview.widget.ListAdapter
 import androidx.recyclerview.widget.RecyclerView
 import com.android.aniviewer_kun.databinding.RowMediaBinding
 import com.android.aniviewer_kun.glide.Glide
 import com.android.aniviewer_kun.type.MediaStatus
+import com.android.aniviewer_kun.ui.Top100Fragment
 
 
-class MediaRowAdapter(private val viewModel: MainViewModel)
+class MediaRowAdapter(private val viewModel: MainViewModel, private val parentFragment: Fragment)
     : ListAdapter<MediaListQuery.Medium?, MediaRowAdapter.VH>(RedditDiff()) {
 
     private var media = listOf<MediaListQuery.Medium?>()
 
     inner class VH(val binding: RowMediaBinding): RecyclerView.ViewHolder(binding.root) {
         init {
-            binding.romajiTitle.setOnClickListener {
+            binding.root.setOnClickListener {

```

Comment:
I changed this so you can click on the entire card to bring up details. I think that would be simpler to the user.